databrowser/src/layouts/menu/DatasetMenu.vue

```diff
@@ -0,0 +1,102 @@
+<template>
```

Comment:
I would not call this component `DatasetMenu`, because it can also be a menu with something else. The underlying view logic should be decoupled from the content that is shown. The datasets should just be the content.